lib/SILOptimizer/Mandatory/DIMemoryUseCollector.h

```diff
@@ -352,6 +357,15 @@ struct DIElementUseInfo {
 void collectDIElementUsesFrom(const DIMemoryObjectInfo &MemoryInfo,
                               DIElementUseInfo &UseInfo);
 
+/// Check whether given function is a user-defined initializer of a
+/// type wrapped type.
+bool canHaveTypeWrapperLocalStorageVar(SILFunction &F);
```

Comment:
is there a situation where the user-defined init is for a type-wrapper and it does **not** have a local storage var inside of it? I guess that can happen if storage is empty? If that's the case, here's a suggested edit:

```suggestion
/// Check whether given function is a user-defined initializer of a
/// type wrapped type. Does not guarantee it contains a storage var.
bool canHaveTypeWrapperLocalStorageVar(SILFunction &F);
```

Reply:
`_storage` is always injected by type-checker even if it’s `()` to avoid having to worry about it’s presence.

Reply:
Ah ok thanks! then all I'd suggest is perhaps calling this function `hasTypeWrapperLocalStorageVar`

Reply:
Sure! I'll amend the comment that introduced it.

Reply:
✅